src/main/java/org/junit/rules/RuleChain.java

```diff
@@ -72,7 +72,7 @@ private RuleChain(List<TestRule> rules) {
     }
 
     /**
-     * Create a new {@code RuleChain}, which encloses the {@code nextRule} with
+     * Create a new {@code RuleChain}, which encloses the {@code enclosedRule} with
```

Comment:
I slightly prefer "...encloses the given {@link Rule} with..." (Javadoc should read like sentences).